Sources/StreamChat/APIClient/Endpoints/ModerationEndpoints.swift

```diff
@@ -16,6 +16,44 @@ extension Endpoint {
     }
 }
 
+// MARK: - User blocking
+
+extension Endpoint {
+    static func blockUser(_ userId: UserId, keepChannelsVisible: Bool = false) -> Endpoint<BlockedUsersResponse> {
```

Comment:
Preferrable, we only set default values in the controller level. Not on the endpoints or updaters. This is to make sure that we don't forget to fill in all the arguments. Only the in the controller level we should provide default arguments to simplify the customers API 👍 
```suggestion
    static func blockUser(_ userId: UserId, keepChannelsVisible: Bool) -> Endpoint<BlockedUsersResponse> {
```

Reply:
Updated 👍